app/components/buttons/text-button.jsx

```
@@ -9,7 +10,7 @@
  */
 export default function TextButton(props) {
     return(
-        <div className="text-btn" role='button' onClick={props.onClick}>
+        <div className={twMerge("text-btn", props.className)} role='button' onClick={props.onClick}>
```

Biome (1.9.4)

[error] The HTML element with the interactive role 'button' is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like 'button'.
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)